### PR TITLE
Updated design guidelines

### PR DESCRIPTION
**Pull request by mel-manohar**
The design section in docs will now has 3 subsections: UI Kit, Guidelines, Docker's Design Principles

### CODE REVIEW

File: docs/design/design-guidelines.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (benja-M-1) [inline docs/design/design-guidelines.md:81]: nit: we can remove those lines

**Review by benja-M-1 — CHANGES_REQUESTED** [review]
@mel-manohar I think you can get rid of the `<br>` and just add a carriage return instead. The markdown parser should replace them with a `<br/>` itself.

File: docs/design/design-guidelines.md (hunk 1 above)
Comment (mel-manohar) [reply docs/design/design-guidelines.md]: Removed

**Comment by mel-manohar** [comment]
@benja-M-1 when I preview it in (VSCode) without the <br> tags, the carriage return does not seem to automatically include the line break. Should I still remove it ?

**Comment by mel-manohar** [comment]
> Do you mean the content all appears on one line, or that there isn't enough space between the lines (so you wanted a blank one to pad it out further). If the former, something weird is happening. If the latter, we should defer to the docs site's stylesheet (which VS Code's preview won't reflect).

It was the latter. I have removed the <br> now.

**Comment by mark-dr** [comment]
Was this deliberately closed, @mel-manohar? My PR (#158) was based on it :sweat_smile:

**Comment by mel-manohar** [comment]
@mark-dr by mistake sorry Mark. I meant to simply comment. Re-opened now.

File: docs/design/design-guidelines.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+# Design Guidelines
+
+At Docker, we aim to build tools that integrate into a user's existing workflows rather than requiring them to adopt new ones. To that end, we strongly recommend that you follow these guidelines when building extensions. We will be reviewing and approving your Marketplace publication based on these requirements.
+
+
+## Build a consistent experience with Docker Desktop.
+Use the [Docker Material UI Theme](https://www.npmjs.com/package/@docker/docker-mui-theme) and the [Docker Extensions Styleguide](https://www.figma.com/file/U7pLWfEf6IQKUHLhdateBI/Docker-Design-Guidelines?node-id=1%3A28771) to ensure that your extension feels like it is part of Docker Desktop to create a seamless experience for users.
+
+- Ensure the extension has both a light and dark theme. Using the components and styles as per the Docker styleguide will ensure that your extension meets the [level AA accessibility standard.](https://www.w3.org/WAI/WCAG2AA-Conformance)
```
Comment (aevesdocker) [inline docs/design/design-guidelines.md:15]:
```suggestion
- Ensure the extension has both a light and dark theme. Using the components and styles as per the Docker style guide ensures that your extension meets the [level AA accessibility standard.](https://www.w3.org/WAI/WCAG2AA-Conformance)
```

File: docs/design/ui-kit.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+---
+title: Docker extension design
+description: Docker extension design
+keywords: Docker, extensions, design
+---
+
+## Docker UI Kit
+
+Our Design System is currently being developed. In the meantime, you will find some basic styles (color, typography) and components in this [Docker Extensions Styleguide](https://www.figma.com/file/U7pLWfEf6IQKUHLhdateBI/Docker-Design-Guidelines?node-id=1%3A28771). Using the components and styles as per the Docker Extensions Styleguide will ensure that your extension meets the [level AA accessibility standard.](https://www.w3.org/WAI/WCAG2AA-Conformance)
```
Comment (aevesdocker) [inline docs/design/ui-kit.md:9]:
```suggestion
Our Design System is currently being developed. In the meantime, you will find some basic styles (color, typography) and components in this [Docker Extensions Styleguide](https://www.figma.com/file/U7pLWfEf6IQKUHLhdateBI/Docker-Design-Guidelines?node-id=1%3A28771). Using the components and styles as per the Docker Extensions Styleguide, ensures your extension meets the [level AA accessibility standard.](https://www.w3.org/WAI/WCAG2AA-Conformance)
```

**Review by aevesdocker — APPROVED** [review]
Some nit-picky tweaks @mel-manohar

**Comment by mel-manohar** [comment]
Thank you so much @aevesdocker

File: docs/design/ui-kit.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+title: Docker extension design
+description: Docker extension design
+keywords: Docker, extensions, design
+---
+
+## Docker UI Kit
+
+Our Design System is currently being developed. In the meantime, you will find some basic styles (color, typography) and components in this [Docker Extensions Styleguide](https://www.figma.com/file/U7pLWfEf6IQKUHLhdateBI/Docker-Design-Guidelines?node-id=1%3A28771). Using the components and styles as per the Docker Extensions Styleguide, ensures your extension meets the [level AA accessibility standard.](https://www.w3.org/WAI/WCAG2AA-Conformance)
+
+<br>
```
Comment (benja-M-1) [inline docs/design/ui-kit.md:11]:
```suggestion
```

File: docs/design/design-principles.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+We seek out moments of purposeful delight that elevate rather than distract, making work easier and more gratifying. Simple tasks are automated and users are left with more time for innovation.
+<br>
+
+## Build trust through transparency.
+We always provide clarity on what is happening and why. No amount of detail is withheld; the right information is shown at the right time and is always accessible.
+<br>
+
+## Scale with intention.
+Our products focus on inclusive growth and are continuously useful and adapt to match changing individual needs. We support all levels of expertise by meeting users where they are with conscious personalization.
+<br>
```
Comment (benja-M-1) [inline docs/design/design-principles.md:27]:
```suggestion
```

File: docs/design/design-principles.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+People from all levels of experience should feel they know how to use our product. Experiences are familiar, unified, and easy to use so all users feel like experts.
+<br>
+
+## Infuse productivity with delight.
+We seek out moments of purposeful delight that elevate rather than distract, making work easier and more gratifying. Simple tasks are automated and users are left with more time for innovation.
+<br>
+
+## Build trust through transparency.
+We always provide clarity on what is happening and why. No amount of detail is withheld; the right information is shown at the right time and is always accessible.
+<br>
```
Comment (benja-M-1) [inline docs/design/design-principles.md:23]:
```suggestion
```

File: docs/design/design-principles.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+We anticipate needs and provide simple explanations with clear actions so people are never lost and always know what to do next. Recommendations lead users to functionality that enhances the experience and extends their knowledge.
+<br>
+
+## Create value through confidence.
+People from all levels of experience should feel they know how to use our product. Experiences are familiar, unified, and easy to use so all users feel like experts.
+<br>
+
+## Infuse productivity with delight.
+We seek out moments of purposeful delight that elevate rather than distract, making work easier and more gratifying. Simple tasks are automated and users are left with more time for innovation.
+<br>
```
Comment (benja-M-1) [inline docs/design/design-principles.md:19]:
```suggestion
```

File: docs/design/design-principles.md [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+
+# Docker's Design Principles
+
+## Provide actionable guidance.
+We anticipate needs and provide simple explanations with clear actions so people are never lost and always know what to do next. Recommendations lead users to functionality that enhances the experience and extends their knowledge.
+<br>
+
+## Create value through confidence.
+People from all levels of experience should feel they know how to use our product. Experiences are familiar, unified, and easy to use so all users feel like experts.
+<br>
```
Comment (benja-M-1) [inline docs/design/design-principles.md:15]:
```suggestion
```

File: docs/design/design-principles.md [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+title: Docker design principles
+description: Docker extension design
+keywords: Docker, extensions, design
+---
+
+# Docker's Design Principles
+
+## Provide actionable guidance.
+We anticipate needs and provide simple explanations with clear actions so people are never lost and always know what to do next. Recommendations lead users to functionality that enhances the experience and extends their knowledge.
+<br>
```
Comment (benja-M-1) [inline docs/design/design-principles.md:11]:
```suggestion
```

**Review by benja-M-1 — CHANGES_REQUESTED** [review]
@mel-manohar I add more `<br>` removals :) After that it should be good :+1:

**Review by gtardif — APPROVED** [review]

**Comment by gtardif** [comment]
Fixed image links / size after jekyll docs generation (for docs.docker.com)

Screenshot 2022-05-10 at 12 03 30

**Pull request closed without merging**